src/main_operations/modules/apriltags/pre_processing/ai_acceleration/training/test_models/test_position_predictor_video.py

```
@@ -0,0 +1,409 @@
+        model, device = load_model(model_path, device=device)
+
+        # Process video
+        process_video(
+            video_path=video_path,
+            model=model,
+            device=device,
+            output_path=output_path,
+            confidence_threshold=confidence_threshold,
+        )
+
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)